File: pilhas-filas/javascript/stack1-test.js
import Stack1 from "./stack1.js";

let stack = new Stack1(10);

console.log("stack.count(): " + stack.count())
console.log("stack.isEmpty(): " + stack.isEmpty());

stack.push("Maria");
stack.push("Joao");
stack.push("Ana");
console.log("Adicionados: Maria, Joao, Ana");

console.log("stack.count(): " + stack.count())
console.log("stack.isEmpty(): " + stack.isEmpty());
console.log("stack.peek(): " + stack.peek());

console.log("Itens:");
while (!stack.isEmpty()) {
  const item = stack.pop();
  console.log(item);
}

File: pilhas-filas/python/stack1_test.py
from stack1 import Stack1

stack = Stack1(10)

print(f"stack.count(): {stack.count()}")
print(f"stack.is_empty(): {stack.is_empty()}")

stack.push("Maria")
stack.push("Joao")
stack.push("Ana")
print("Adicionados: Maria, Joao, Ana")

print(f"stack.count(): {stack.count()}")
print(f"stack.is_empty(): {stack.is_empty()}")
print(f"stack.peek(): {stack.peek()}")

print("Itens:")
while not stack.is_empty():
    print(stack.pop())
